tests/install_upgrade_operators/role_aggregation/test_role_aggregation_rbac_enforcement.py

```
@@ -46,29 +64,73 @@ def test_vm_list_forbidden_when_aggregation_disabled(self, role):
+        cluster_role = ClusterRole(name=role, client=admin_client, ensure_exists=True)
+        with RoleBinding(
+            name=f"test-role-bind-{role}",
+            namespace=namespace.name,
+            client=admin_client,
+            subjects_kind="User",
+            subjects_name=UNPRIVILEGED_USER,
+            subjects_namespace=namespace.name,
+            role_ref_kind=cluster_role.kind,
+            role_ref_name=cluster_role.name,
+        ):
+            LOGGER.info(f"Baseline: waiting for {role} role RBAC propagation before testing")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)